test/cases/basics.ts

```diff
@@ -43,6 +43,21 @@ export var configurations = [
   },
   {
     settings: {
+      useInterfaceDeclaration: true,
+      exportInterfaces: true
```

Comment:
Let's always export and get rid of this option - everything is public in a JSON schema, so let's keep it that way here.